index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <title>Title</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
   </head>
   <body>
-    <h1>Моя первая страница</h1>
-    <a href="./about/index.html">Об авторе</a>
+    <h1 class="heading">What do we offer</h1>
+    <div class="line"></div>
+    <img class="images" src="triphouse-img/vector1.png" alt="Support 24/7" />
+    <img class="images" src="triphouse-img/vector2.png" alt="Communicate directly"/>
+    <img class="images" src="triphouse-img/vector3.png" alt="Book online" />
```

Comment:
also, create div wrapper for your image and text. So your image in description will be located in one conteainer and you can align it one below other. and then this wrappers can be alligned in th main container
<div class= "item_wrapper">
<img href="some path" alt=" some desc"/>
<p> Some text</p>
</div>

in css
.item_wrapper {
some properties
}